### DISCUSSION TITLE
Module Not Showing Up After Installation (Odoo14)

### DISCUSSION THREAD

**Discussion by it12uw** [OCA/odoo-community.org#108] [Q&A]
I just create my costume module in Odoo14 and when i install the module and success, the module not showing up in my sidebar.
i already try to update all and refresh the browser but the module still not showing up.

is there any problem why the module not showing up in odoo sidebar?
or maybe you guys have any solution for this case.

here is my __manifest__.py and ir.model.access.csv
**__manifest__.py**
`
{
    'name': 'Costume Snap QC',
    'version': '1.0',
    'category': 'Manufacture',
    'sequence': 6,
    'summary': 'Costume Module For Snap QC',
    'author': 'Steven Morison, '
              'stevenmorizon123@gmail.com',
    'description': """

This is a module for Snap QC (Quality Control) which has the following functions:

- check the existing machine based on the blocks.
- check the existing machine based on the blocks.
- check the existing machine based on the blocks.

""",
    'depends': [
        'base',
        'web',
  ],
    'website': '',
    'data': [
        'security/ir.model.access.csv',
        'views/confirm_snap.xml',
        'views/costume_snap_qc.xml',
    ],

    'installable': True,
    'auto_install': False,
    'license': 'LGPL-3',
}
`

**ir.model.access.csv**

`id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
access_costume_snap_qc_user,costume_snap_qc.user,model_costume_snap_qc,,1,1,1,0
access_costume_snap_qc_manager,costume_snap_qc.manager,model_costume_snap_qc,base.group_user,1,1,1,1
`

thankyou

**Comment by jguenat** [comment]
You need to add a menuitem for you app to show in the odoo sidebar.

Here you can find an example for the Contacts menu

`<menuitem name="Contacts"
        id="menu_contacts"
        sequence="4"
        web_icon="contacts,static/description/icon.png"
        groups="base.group_user,base.group_partner_manager"/>`

For newcomers in Odoo I suggest the following book

https://www.packtpub.com/product/odoo-15-development-essentials-fifth-edition/9781800200067
